README.md

```diff
@@ -1,19 +1,25 @@
 # triton-service-groups
 
+Triton Service Groups provide monitoring for Triton applications, automatically scaling to meet capacity needs and maintain performance at the lowest possible cost. Manage applications using the TSG API or with Terraform to scale resources on Triton including VMs and containers. 
+
+Triton Service Groups help you maximize application efficiency so that you only pay for resources that you actually need. If demand decreases, TSG will adjust and remove unnecessary resources to optimize your spending on Triton.
+
+At this time, there is no integration with `triton`.
```

Comment:
You mean with the Triton CLI tools, right?

Reply:
Sophisticated users can likely guess that `triton` is intended to indicate the name of a CLI executable, but people are sloppy with the use of backticks (we use them for quotes, others use them for emphasis, etc.).